### ISSUE TITLE
[Q] How to plot Accuracy (or any metrics) for multiple classes in a single plot?

### ISSUE THREAD

**Issue by vkrishnamurthy11** [wandb/wandb#8005]
For the purpose of this discussion lets assume I am training an MNIST classifier, so 10 classes.
I perform validation after every epoch.

How can I combine the accuracy plots for all 10 classes in a single plot for every epoch that I perform validation? 
I don't want to have to open Wandb after my training procedure and combine 10 different plots together.

I also do not want to be storing the scores of previous epochs and plotting the full table after every validation step. Ideally I am just adding more points to an already existing graph.

My training loop would look something like this:

```
for epoch in range(epochs):
      train_model()
      metrics = validate_model()
     # Here metrics contains the validation accuracy of every single class.
      wandb.log() # Method that can combine the accuracies for all 10 classes into a single plot.

**Comment by paulosabile-wb** [comment]
Hi @vkrishnamurthy11 Good day and thank you for reaching out to us. Could you please try to use an alternative method to sync offline runs like this?
`
wandb sync [PATH] -p <PROJECT_NAME> -e <ENTITY> --id new_run_id`


Let me know if this works for you.


Regards,
Paulo

**Comment by paulosabile-wb** [comment]
Hi @vkrishnamurthy11 

Good day and thank you for reaching out to us. 
 
You could try using [resuming ](https://docs.wandb.ai/guides/runs/resuming)runs if you need to add another metric to your existing graphs. You could try something like below and see if this works for you:
 
```
import wandb
run = wandb.init(project="project-name", id="1234567", resume="allow")
run.log({"loss": 12, "accuracy": 5})
run.finish()
```
Then you can just rerun the same script once you require another metric to appear on the same chart.

Please note that if you pass resume="must"  it will force to resume that run, and will error if the run isn’t found, while allow will create a new run if it’s not found.
 
I hope this helps!
 
 
Thanks,
Paulo

**Comment by paulosabile-wb** [comment]
Hi @vkrishnamurthy11 , since we have not heard back from you we are going to close this request. If you would like to re-open the conversation, please let us know!
